CHANGELOG.rst

```diff
@@ -4,7 +4,7 @@ Changelog
 0.5.10 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Change battery in DeviceSerializer to use dn_battery_pct in device model
```

Comment:
You are describing the code diff, and it would make a bad commit message too.

Rather describe what is changing for users (developers as it's a library) in a changelog. Example: "Improve performance in private vehicle API by removing unnecessary query".

The rest is implementation details.